### PR TITLE
move receiver name outside and add setting to change the color

### PR DESCRIPTION
**Pull request by reytum**
* Moved the receiver name text view outside the message box
* Added setting to change the color of the receiver name text view

### CODE REVIEW

**Comment by devashishmamgain** [comment]
@reytum can you share apk for this

File: app/src/main/assets/applozic-settings.json [hunk 1]
```diff
@@ -154,5 +154,6 @@
     "submit": false,
     "link": false,
     "quickReply": false
-  }
+  },
+  "receiverNameTextColor": "#5C6677"
```
Comment (devashishmamgain) [inline app/src/main/assets/applozic-settings.json:158]: Is it receiver or sender? Name seems confusing

Comment (reytum) [reply app/src/main/assets/applozic-settings.json:158]: Its receiver. It can be agent or Bot for User.

Comment (devashishmamgain) [reply app/src/main/assets/applozic-settings.json:158]: If you send me a message then I m the receiver?

Comment (devashishmamgain) [reply app/src/main/assets/applozic-settings.json:158]: Left side one will be the sender right?

Comment (reytum) [reply app/src/main/assets/applozic-settings.json:158]: Yes. But I think it is easier as receiver as the word is being used in several other settings like receivedMessageTextColor, etc. Maybe we can make it as receivedMessageTitleColor.

Comment (devashishmamgain) [reply app/src/main/assets/applozic-settings.json:158]: receivedMessageTitleColor is better if we have something for message.
This is for name so we can go ahead with existing used

**Review by devashishmamgain — APPROVED** [review]

**Pull request merged**